packages/source-http/src/createHttpSource.ts

```diff
@@ -39,10 +39,11 @@ export interface CreateHttpSourceParams<TResponse, TPage>
  * For use inside *other* sources.
  * Allows a transformer function to be passed directly without the need for dynamic imports.
  *
+ * SourceConfig is also optional here so that scheduling can be ignored.
  */
-export function createHttpSource<TResponse, TPage extends Page>(
+export function createHttpSource<TResponse, TPage = Page>(
```

Comment:
This no longer really creates a source because:

1. it can return things that are not pages
2. it can run unscheduled

Another name needed?  `createHttp$`?

Reply:
does a source have to return a page ?
Maybe it's ok calling it a Source ?

Reply:
I think sources should return pages....but this here can be used to build other sources, that's the difference